pkgs/development/python-modules/psycopg2cffi/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage, cffi, fetchFromGitHub, lib, postgresql, pytest, six }:
+
+buildPythonPackage rec {
+  pname = "psycopg2cffi";
+  version = "2.8.1";
+
+  src = fetchFromGitHub {
+    owner = "Omegapol";
+    repo = pname;
+    rev = "c202b25cd861d5e8f0f55c329764ff1da9f020c0";
+    sha256 = "09hsnjkix1c0vlhmfvrp8pchpnz2ya4xrchyq15czj527nx2dmy2";
+  };
+
+  nativeBuildInputs = [ postgresql ];
+  propagatedBuildInputs = [ six cffi ];
+  checkInputs = [ pytest ];
+
+  checkPhase = ''
```

Comment:
I think `pytestCheckHook` takes care of this

Reply:
Ah, yes :)